cmd/gotohelm/main.go

```diff
@@ -5,27 +5,42 @@ import (
 	"fmt"
 	"os"
 	"path"
+	"strings"
 
 	"github.com/redpanda-data/helm-charts/pkg/gotohelm"
 	"golang.org/x/tools/go/packages"
 )
 
+type dependencies []string
+
+func (d *dependencies) String() string {
+	return fmt.Sprint(*d)
+}
+
+func (d *dependencies) Set(value string) error {
+	for _, dep := range strings.Split(value, ",") {
+		*d = append(*d, strings.TrimSpace(dep))
+	}
+	return nil
+}
+
 func main() {
+	var deps dependencies
```

Comment:
No change needed but given that we accept variadic args here and that `LoadPackages` seems like it would play nice, you could make the invocation: `gotohelm <main package> [deps...]`

The transpile phase could then be:
```go
root = pkgs[0]
deps = pkgs[1:]

chart, err := gotohelm.Transpile(pkg, deps)
```

_In theory_ that might mean you could easily do something like:

```bash
# To be read as "Transpile the redpanda chart. Anything imported by the redpanda chart should be treated like a subchart/dependency.
gotohelm ./charts/redpanda ./charts/...
```

Though that will depend on the exact resolution of `packages.Load`, so play around with it a bit.

Reply:
Agree